PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutVaultExperienceContext.kt

```diff
@@ -0,0 +1,16 @@
+package com.paypal.android.paypalwebpayments
+
+data class PayPalWebCheckoutVaultExperienceContext internal constructor(
```

Comment:
Adding this `PayPalWebCheckoutVaultExperienceContext` type to make constructing the Create Setup Token call easier for merchants. Having the SDK control the format of the return and cancel URLs can also help us enforce an internal deep linking schema for parsing browser switch results in a predictable manner.

Reply:
Can we add a doc string for this?

Reply:
Same for docs. Not sure if we'll end up keeping this. I'd like to see what iOS does to collaborate on the best API.